programs/ssl/ssl_base64_dump.c

```diff
@@ -177,6 +182,14 @@ void print_hex( const uint8_t *b, size_t len )
     fflush(stdout);
 }
 
+void print_time( const time_t *time )
+{
+    char buf[20];
+    static const char * const format = "%G-%m-%d %H:%M:%S";
+    strftime( buf, 20, format, gmtime( time ) );
```

Comment:
DRY (don't repeat yourself). Use sizeof(buf) to keep both places always in sync, and to show that the 20s have the same meaning.